### ISSUE TITLE
Question about the metric reported in the paper?

### ISSUE THREAD

**Issue by dsj96** [google-research/text-to-text-transfer-transformer#1091]
**Question about the metric reported in the paper?.**
HELLO! I am a new NLPer.
1. I would like to know which stage of the model was used for dataset c4(unlabeled), glue(labeled), superGLUE WMT(labeled), respectively?
In paper, section 2.4, I find that
`We instead allow for separately fine-tuning the model on each individual task and use short task prefixes instead of an explicit question-answer format.`
As shown in Table 1 of the paper, dose `T5` model pre-trained on dataset C4, then  fine-tuned on GLUE, CNNDM, SQuAD, SGLUE and WMT dataset, respectively? Finally, reported the score in Table 1.
2. Other Large Language Models, like GPT, GPT2, have these models been fine-tuned on GLUE or superGLUE   before reporting the scores?

Thank you!
